geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/ClientHealthMonitorJUnitTest.java

```diff
@@ -14,242 +14,67 @@
+    ClientProxyMembershipID mockId = mock(ClientProxyMembershipID.class);
+    ServerConnection mockConnection = mock(ServerConnection.class);
 
-  @Test
-  public void monitorIntervalDefaultsWhenInvalidValue() {
-    String monitorInterval = "this isn't a number";
-    System.setProperty(ClientHealthMonitor.CLIENT_HEALTH_MONITOR_INTERVAL_PROPERTY,
-        monitorInterval);
+    clientHealthMonitor.addConnection(mockId, mockConnection);
+    clientHealthMonitor.receivedPing(mockId);
 
-    assertNotNull(ClientHealthMonitor
-        .getInstance(mock(InternalCache.class), 0, mock(CacheClientNotifierStats.class))
-        .getMonitorInterval());
+    Awaitility.await()
+        .atMost(2 * (monitorIntervalMillis + pingIntervalMillis), TimeUnit.MILLISECONDS)
+        .until(() -> verify(mockConnection).handleTermination(true));
   }
 
-  /**
-   * This test performs the following:<br>
-   * 1)create server<br>
-   * 2)initialize proxy object and create region for client<br>
-   * 3)perform a PUT on client by acquiring Connection through proxy<br>
-   * 4)stop server monitor threads in client to ensure that server treats this as dead client <br>
-   * 5)wait for some time to allow server to clean up the dead client artifacts<br>
-   * 6)again perform a PUT on client through same Connection and verify after the put that the
-   * Connection object used was new one.
-   */
-  @Test
-  public void testDeadClientRemovalByServer() throws Exception {
-    System.setProperty(ClientHealthMonitor.CLIENT_HEALTH_MONITOR_INTERVAL_PROPERTY,
-        String.valueOf("100"));
-    PORT = createServer();
-    createProxyAndRegionForClient();
-    StatisticsType statisticsType = this.system.findType("CacheServerStats");
-    final Statistics statistics = this.system.findStatisticsByType(statisticsType)[0];
-    assertEquals(0, statistics.getInt("currentClients"));
-    assertEquals(0, statistics.getInt("currentClientConnections"));
-    this.system.getLogWriter()
-        .info("beforeAcquireConnection clients=" + statistics.getInt("currentClients") + " cnxs="
-            + statistics.getInt("currentClientConnections"));
-    Connection connection1 = proxy.acquireConnection();
-    this.system.getLogWriter()
-        .info("afterAcquireConnection clients=" + statistics.getInt("currentClients") + " cnxs="
-            + statistics.getInt("currentClientConnections"));
-    this.system.getLogWriter().info("acquired connection " + connection1);
-
-    Awaitility.await().pollDelay(0, TimeUnit.MILLISECONDS).pollDelay(10, TimeUnit.MILLISECONDS)
-        .atMost(1, TimeUnit.SECONDS).until(() -> statistics.getInt("currentClients") == 1);
 
-    assertEquals(1, statistics.getInt("currentClients"));
-    assertEquals(1, statistics.getInt("currentClientConnections"));
-    ServerRegionProxy srp = new ServerRegionProxy("region1", proxy);
-
-    srp.putOnForTestsOnly(connection1, "key-1", "value-1", new EventID(new byte[] {1}, 1, 1), null);
```

Comment:
Check to see if this method is no longer used & if so delete it.

Reply:
This code wasn't actually removed, this test still exists in the IntegrationTest.